skeleton/backstage/template.yaml

```diff
@@ -113,10 +115,12 @@ spec:
                   enum:
                     - tekton
                     - jenkins
+                    - azure
```

Comment:
Let's not add this as an option to GitLab. From the Azure UI, there doesn't seem to be an option to do this which seems to indicate it's not really supported.

Reply:
![image](https://github.com/user-attachments/assets/646597e9-4fbb-4282-acdc-0d1abba9656c)

@lcarva I see what you mean, I have removed the option from the GitLab SCM choice to address the technology limitation on Azure.